websockify/websocketproxy.py

```diff
@@ -222,6 +222,18 @@ def do_proxy(self, target):
                 tqueue.extend(bufs)
 
                 if closed:
+
+                    while (len(tqueue) != 0):
+                        # Send queued client data to the target
+                        dat = tqueue.pop(0)
+                        sent = target.send(dat)
```

Comment:
Ideally, this should be a `select()` loop, but since the socket seems to be blocking this should work anyway.